README.md

```diff
@@ -44,5 +37,3 @@ Give a ⭐️ if you like this project!
 - etc
 
 ## 📝 License
-
```

Comment:
- [OPTIONAL] Please it will be nice to add a link to the license file, as this will make your readme file more informative and professional.